spec/spec_helper.rb

```diff
@@ -1,4 +1,20 @@
 require 'pathname'
+
+require 'simplecov'
+
+SimpleCov.start do
+  track_files 'lib/**/*.rb'
+  add_filter 'lib/reek/version.rb' # version.rb is loaded too early to test
+end
+
+if ENV['FAIL_ON_LOW_COVERAGE'] == '1'
+  SimpleCov.at_exit do
+    SimpleCov.result.format!
+    SimpleCov.minimum_coverage 98.71
```

Comment:
> Seems having these values will lead to a lot of churn of this file.

Possibly, but then again I'm not going to be watching this number very closely and update it every time our coverage increases by 0.01% :laughing:.

My main goal with this is to be able to quickly spot parts of the code that are not used anymore. If an external runner were available (e.g., `simplecov rspec spec`), I'd use that instead.